lapis/job.py

```diff
@@ -87,6 +94,10 @@ def waiting_time(self) -> float:
             return self.in_queue_until - self.in_queue_since
         return float("Inf")
 
+    @property
+    def has_inputfiles(self) -> bool:
+        return bool(self.inputfiles)
+
```

Comment:
You can drop this methods. You already have the field `inputfiles`. So when having a construct `if job.inputfiles: pass` you are already good to go :)